Resources/Prototypes/Entities/Objects/Weapons/Melee/spear.yml

```diff
@@ -48,7 +48,7 @@
   - type: IncreaseDamageOnWield
     damage:
       types:
-        Piercing: 7
+        Piercing: 6
```

Comment:
i might misunderstand this, but doesn't this decrease the spears damage from before from 22 to 21?

Reply:
oops